isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -36,7 +36,10 @@
       adjustment.  The initial states of the points are tagged as <def>a priori</def> in the
       <def>control network</def>, and their final states are tagged as <def>adjusted</def>.
       Updated camera information is only written to the <def>cube</def> labels if the bundle
-      converges and the <b>UPDATE</b> parameter is selected.
+      converges and the <b>UPDATE</b> parameter is selected. In addition to <b>UPDATE</b> being
+      true and bundle converging, if the <b>ADJUSTMENT_INPUT</b> is set to an HDF5 file containing 
+      bundle adjustment values (usually the HDF5 output file from <b>ADJUSTMENTOUT_H5</b>), those 
+      values can be applied to the <def>cube</def> labels.
```

Comment:
I would go more into detail about how this file is formatted in case someone wants to open it outside of the context of this app.